src/openrct2/actions/TrackPlaceAction.cpp

```diff
@@ -522,7 +522,7 @@ GameActions::Result TrackPlaceAction::Execute() const
             supportHeight = (10 * COORDS_Z_STEP);
         }
 
-        cost += ((supportHeight / (2 * COORDS_Z_STEP)) * ride->GetRideTypeDescriptor().BuildCosts.SupportPrice) * 5;
+        cost += (supportHeight / (2 * COORDS_Z_STEP)) * ride->GetRideTypeDescriptor().BuildCosts.SupportPrice;
```

Comment:
This looks incorrect - why did you remove the `* 5` part?

Reply:
The previous cost was calculated as
`canBuild.Cost + price / 2 * 10;`
which is
`canBuild.Cost + (price / 2 * 10);`
so I assumed that the `* 5` in this line was added to offset too low costs introduced by this parenthesis bug. I have not checked the original disassembly though so maybe it should be something yet different, but looking at what @duncanspumpkin found previously I am almost 100% sure that the algo should be `((canBuild.Cost + price) / 2) * 10;`. Maybe costs are too high for a reason different than this `* 5`?

Reply:
![image](https://user-images.githubusercontent.com/1277401/150689651-661640e6-e069-4459-946a-b8b6b2c28d77.png)
This is the original code. dl is the supportHeight / Coords_Z_step. So it is doing `cost += (dl / 2) * supportPrice` Your change looks correct.